operations/openstack/use-swift-blobstore.yml

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
Even though this is ops-file is IaaS-specific, it's not required for deploying to Openstack. Maybe just move this one directory up -- `operations/use-swift-blobstore.yml`?

Reply:
You are correct. In theory I could imagine people installing *just swift* to get an on-premise blobstore.